app/models/user.rb

```diff
@@ -1,12 +1,9 @@
 class User < ApplicationRecord
-  validates :name, presence: true
-  validates :posts_counter, numericality: { greater_than_or_equal_to: 0 }
-
   has_many :comments, foreign_key: :author_id
   has_many :posts, foreign_key: :author_id
   has_many :likes, foreign_key: :author_id
 
   def last_three_posts
-    posts.includes(:author).order(created_at: :desc).limit(3)
+    Post.last(3)
   end
```

Comment:
- [ ] When I open a user with 0 posts I still can see posts listed on users show page: 
<img width="650" alt="Screenshot 2022-08-19 at 14 44 40" src="https://user-images.githubusercontent.com/77038610/185602229-0df13b3a-3ccb-4776-8cac-60f84020a905.png">

This is happening because last_three_posts is getting data from another model, not relation. So instead of 
`Post.last(3)` you should be `using posts.last(3)` in this case the query will target only the posts where user is author 😉